src/main/java/pl/sda/eventmanager/services/AdminPanelService.java

```diff
@@ -0,0 +1,24 @@
+package pl.sda.eventmanager.services;
+
+import org.springframework.stereotype.Service;
+import pl.sda.eventmanager.model.User;
+import pl.sda.eventmanager.repositories.UserRepository;
+
+import java.util.List;
+
+@Service
+public class AdminPanelService {
```

Comment:
Better name would be `AdminOperationsService`/`AdminActionsService`, provided that you will keep there methods that are **ONLY** performed by admin and are not performed anywhere else